fuzz_lightyear/generator.py

```diff
@@ -140,6 +140,8 @@ def _generate_request_graph() -> Dict[str, set]:
         for operation_id in dir(getattr(client, tag_group)):
             operation = getattr(getattr(client, tag_group), operation_id).operation
             responses = operation.op_spec.get('responses', {}).get('200', {})
+            if not responses:
```

Comment:
Im just suggesting this cause 200 and 201 are not happening the same time!
```suggestion
           responses = operation.op_spec.get('responses', {})
           responses = responses.get('200', {}) or  responses.get('201', {})
```

Reply:
Looks good. I'll include it with future changes 👍🏼